content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -525,21 +525,23 @@ This means that support is still available but will be removed in a future Kuber
 Older versions of Kubernetes also supported the following in-tree PersistentVolume types:
 
 * [`awsElasticBlockStore`](/docs/concepts/storage/volumes/#awselasticblockstore) - AWS Elastic Block Store (EBS)
-  (**not available** in v1.27)
+  (**not available** starting with v1.27)
```

Comment:
I think we should actually distinguish between volume plugins that are supported with CSI Migration vs plugins that are outright removed.

Plugins that are supported with CSI migration allows users to continue using the API and having all requests routed to the CSI driver instead of the intree plugin.

Reply:
Is there a definitive list anywhere of which plugins have a migration path and which do not?

Reply:
According to https://kubernetes.io/docs/concepts/storage/volumes/#migrating-to-csi-drivers-from-in-tree-plugins, it is scattered around the "Volume types" section, but we appear to have removed all the documentation for the GA + removed plugins.

Otherwise, here is the list: https://kubernetes.io/blog/2022/09/26/storage-in-tree-to-csi-migration-status-update-1.25/#timeline-and-status

Reply:
The info in the blog https://kubernetes.io/blog/2022/09/26/storage-in-tree-to-csi-migration-status-update-1.25/#timeline-and-status needs to be updated though. For example, Ceph RBD and Ceph FS in-tree plugins are deprecated and are no longer going through CSI Migration.

We provide an update in SIG Storage session at KubeCon every time so this is the latest:
https://github.com/kubernetes/website/pull/44367#discussion_r1427097893